style.css

```diff
@@ -117,3 +117,439 @@ nav .nav-icon {
+  margin: 80px 24px 0 24px;
+}
+
+.recent-works h2 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round'sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: #091e42;
+  margin: 24px 24px 0 24px;
+}
+
+.recent-works p {
+  width: 327px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin: 12px 24px 0 24px;
+}
+
+.recent-works ul {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  margin: 12px 54px 0 24px;
+  padding: 0;
+}
+
+.recent-works li {
+  background: #fff;
+  border: 1px solid #8993a4;
+  margin-right: 12px;
+  padding: 0;
+}
+
+.recent-works .css {
+  align-items: center;
+  padding: 10px 12px;
+  width: 25px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.recent-works .html {
+  width: 32px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  align-items: center;
+}
+
+.recent-works .boostrap {
+  width: 71px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  align-items: center;
+}
+
+.recent-works .ruby {
+  width: 37px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  align-items: center;
+}
+
+.recent-works button {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  margin: 12px 0 0 24px;
+}
+
+.recent-works .card-works {
+  margin: 24px 24px 0 24px;
+  min-height: 386px;
+  background:
+    linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%),
+    url(images/Snapshoot\ Portfolio.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.card-works h1 {
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding: 84px 16px 0 16px;
+}
+
+.card-works p {
+  width: 295px;
+  height: 87px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  padding: 12px 16px 0 16px;
+}
+
+.card-works ul {
+  padding: 12px 0 0 16px;
+}
+
+.card-works li {
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.card-works button {
+  width: 327px;
+  height: 48px;
+  margin: 22px 24px 0 0;
+  padding: 0;
+}
```

Comment:
Please, adjust the width of the button to match that of the card or vice versa in order to follow the design template.

Also, kindly adjust the border too as the design has no black borders.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199914956-ab116248-38b9-4326-8197-29d7cc5f0db2.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/199915039-31043866-25b6-4f2a-9ddc-b87ac7c4d50f.png)

Reply:
sir it fit perfectly with a screen dimension of 375px. so please i beg your padding since is a continues project it will get fix with other screen size so please padding me with this